docs/API_reference.md

```diff
@@ -275,6 +276,23 @@ There are currently two supported output formats: 'json' and 'dataframe'.
 The 'json' format simply returns the raw response, without page information and with multiple pages combined into a single data structure.  
 The 'dataframe' format transforms that data structure into a pandas DataFrame, which contains subparts in json, that can be split further by the user application.
 
+---
+
+##### plenty_api_get_property_selections: <a name='get-property-selections'></a>
+
+Return a mapping of Selection values for selction properties.
+
+[*Optional parameter*]:
+
+The **refine** field can be used to reduce the request to specific property IDs.
+example: refine={'propertyID': 123}
```

Comment:
So the syntax for multiple IDs is: `{'propertyID': [123, 234]}` ? Or is this only possible for one value? If the latter is the case then please reword the sentence above accordingly. If the former is correct, then please add an additional example highlighting how to work with multiple IDs.